searx/settings.yml

```diff
@@ -21,13 +21,6 @@ search:
   # Default search language - leave blank to detect from browser information or
   # use codes from 'languages.py'
   default_lang: ""
-  # Available languages
-  # languages:
```

Comment:
This option still exists AFIK : https://github.com/searxng/searxng/blob/b14ed494fb70f976cd77c47d27f29e6fb0bac670/searx/settings_defaults.py#L158

Reply:
Ooops .. I will fix it / sorry for merging without review .. / I thought it is OK in context of https://github.com/searxng/searxng/pull/998#pullrequestreview-915363639 .. but it wasn't OK.